### DISCUSSION TITLE
Bind variables don't work in plpgsql blocks

### DISCUSSION THREAD

**Discussion by rock-it-rob** [dbeaver/dbeaver#18257] [Q&A] [answered]
### Description

In DBeaver community version 21.2.4 the following plpgsql code block would prompt for the value of the bind variable. As of the latest version it generates an error.

```
do $$
begin
  raise info '%', :var;
end; $$ language plpgsql;
```

### DBeaver Version

Community 22.2.4

### Operating System

Windows 10

### Database and driver

Postgresql

### Steps to reproduce

Execute the code in the description.

**Comment by E1izabeth** [comment, marked as answer]
Please, check that the setting `Enable parameters in DDL and $$..$$ blocks` is turned on.
![image](https://user-images.githubusercontent.com/28875055/202419656-c828f819-612a-4d02-9c7c-e543db6c2351.png)

**Comment by rock-it-rob** [comment]
That was it!

Thanks
